code_eval.py

```diff
@@ -91,14 +92,26 @@ def python3(cmd):
 
 def prep_input(args):
 	args = args.lstrip()
-	if args.startswith('```') and args.endswith('```'):
-		split = args[3:].split('\n', 1)
+	if args.startswith('```'):
+		code = args[3:].split('```')[0] # ignore anything after the ending ``` (aka discord pings -- @raylu#1337)
+		split = code.split('\n', 1) # to evalute each line independently.
 		if len(split) == 2:
-			language, other_lines = args[3:].split('\n', 1)
+			split = split.split()
```

Comment:
I added a test for `prep_input` here: cdd17abef85e740b5ed4806b5ca1255f89a69f20

this is failing it because you're calling `.split` on a list

Reply:
d'oh, commits submitted.